lesson-2/src/pages/layout/Contants.js

```diff
@@ -0,0 +1,7 @@
+export const HOME = '/';
+export const LOGIN = '/login';
+export const NOT_FOUND = '/403';
+export const SHOP = '/shop';
+export const BOOKS = '/books';
+export const LOGOUT = '/logout';
+export const BOOKLIST = 'http://www.mocky.io/v2/5e3d13222d00005100d95a98';
```

Comment:
This name is a bit misleading. Is it possible to change to perhaps BOOKS_ENPOINT or BOOKS_URL?